package.json

```diff
@@ -41,5 +41,8 @@
   },
   "peerDependencies": {
     "cypress": ">=3.8.1"
+  },
+  "devDependencies": {
+    "cypress": ">=3.8.1"
```

Comment:
needed to be able to reference the built-in Cypress types (in `src/installLogsPrinter.d.ts`).